### PR TITLE
[#6462mB] adding vote count to map mB

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

File: adhocracy4/maps/templatetags/maps_tags.py [hunk 1]
```diff
@@ -31,6 +32,8 @@ def get_points(items):
             positive_rating_count = item.positive_rating_count
         if hasattr(item, "negative_rating_count"):
             negative_rating_count = item.negative_rating_count
+        if hasattr(item, "vote_count"):
+            vote_count = item.vote_count
```
Comment (Rineee) [inline adhocracy4/maps/templatetags/maps_tags.py:36]: this needs to be 
```
        if hasattr(item, "token_vote_count"):
            vote_count = item.token_vote_count
```
as the annotation is named like that

File: adhocracy4/maps/static/a4maps/a4maps_display_points.js [hunk 2]
```diff
@@ -105,9 +106,29 @@ function init () {
...
+    }
+
     function getCommentCount (feature) {
       return '<span class="map-popup-comments-count">' +
                feature.properties.comments_count + ' <i class="far fa-comment" aria-hidden="true"></i>' +
+               '<span class="visually-hidden">' +
+                  django.ngettext(
+                    'person commented on this proposal.',
+                    'persons commented on this proposal.',
+                    feature.properties.vote_count
```
Comment (Rineee) [inline adhocracy4/maps/static/a4maps/a4maps_display_points.js:129]: comments_count

**Comment by Rineee** [comment]
@phillimorland Found two little things, but apart from that good to merge!

File: adhocracy4/maps/templatetags/maps_tags.py (hunk 1 above)
Comment (phillimorland) [reply adhocracy4/maps/templatetags/maps_tags.py:36]: ahhhh right, that makes sense :)

**Comment by Rineee** [comment]
@phillimorland I'll fix the test!

**Review by Rineee — APPROVED** [review]
Yes, cool!

**Pull request merged**